index.html

```diff
@@ -222,30 +260,31 @@ <h3>Skills</h3>
                     <li>Gitlab</li>
                     <li>Terminal</li>
                 </ul>
+            </div>
             </ul>
             </div>
         </section>
-        <section class="contact">
-            <div>
+        <section class="contact" id="Contact">
+            <div class="contact-content">
             <h2>I'm always interested in hearing about new projects, 
                 so if you'd like to chat please get in touch.
             </h2>
             <form action="https://formspree.io/f/xeqdywdz" method="post">
-                <input required type="text" placeholder="Full name" maxlength="30">
-                <input required type="email" placeholder="Email address" name="email">
+                <input required type="text" placeholder="Full name" maxlength="30" class="hide">
+                <input required type="email" placeholder="Email address" name="email" class="email">
                 <textarea required class="inputfield" type="text" placeholder="Enter text here" maxlength="500"></textarea>
                 <button type="submit">Get in touch</button>
```

Comment:
- [optional] Please work more on your contact form section, I could see that the items are not properly aligned according to the given template. 🙏